packages/input/src/index.vue

```diff
@@ -10,7 +10,8 @@
         'el-input-group--append': $slots.append,
         'el-input-group--prepend': $slots.prepend,
         'el-input--prefix': $slots.prefix || prefixIcon,
-        'el-input--suffix': $slots.suffix || suffixIcon || clearable || showPassword
+        'el-input--suffix': $slots.suffix || suffixIcon || clearable || showPassword,
+        'el-input--suffix--passwordClear': clearable && showPassword
```

Comment:
'el-input--suffix--password-clear' looks better?

Reply:
Thanks for your sugesstion, I will change it